website/docs/d/outputs.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_outputs"
+sidebar_current: "docs-datasource-tfe-state-outputs"
+description: |-
+  Get output values from another organization/workspace.
+---
+# Data Source: tfe_outputs
+
+This data source is used to retrieve the state outputs for a given workspace.
+It enables output values in one Terraform configuration to be used in another.
+
+The outputs retrieved from this data source may contain sensitive information.
+To that end, we defaulted to setting the `values` attribute — which contains the
+output data — of this data source to be marked as
+[sensitive()](https://www.terraform.io/docs/language/functions/sensitive.html).
+This means that one must use `nonsensitive()` to display the output value.
+
+## Example Usage
+
+Using the `tfe_outputs` data source, the outputs `foo` and `bar` can be used as seen below:
+
+In the example below, assume we have outputs defined in an my-org/my-workspace:
+
+```
+output "foo" {
+  value = "a"
+}
+
+output "bar" {
+  value = "b"
+}
+```
+
+The `tfe_outputs` data source can now use `foo` and `bar`
+dynamically as seen below.
+
+```hcl
+data "tfe_outputs" "foobar" {
+  organization = "my-org"
+  workspace = "my-workspae"
+}
+
+output "hello" {
+	value = data.tfe_outputs.foobar.values.foo
+}
+
+output "world" {
+	value = data.tfe_outputs.foobar.values.bar
+}
```

Comment:
You may want to use a different example, as 'declaring an output value from an output value' is more complicated than I thought, QAing this 😬

Basically, I didn't know this was a thing:

![image](https://user-images.githubusercontent.com/2430490/130543275-59a4ed5a-8da6-40c9-8436-a216c64d5495.png)

If you declare `sensitive = true` here, it then allows the output to be completed and present in the state file - but the CLI output is still sensitive (`<sensitive>`). So to both successfully output the values and show it in the output requires `nonsensitive()` as well. 

Given that using both of those starts to make the example look overly complicated and more about sensitivity than the actual usage of an output value from one workspace here, I'd consider doing something like...

```hcl
data "tfe_outputs" "foo" {
  organization = "my-org"
  workspace = "my-workspace"
}

resource "random_id" "vpc_id" {
  keepers = {
    # Generate a new ID any time the value of 'bar' in workspace 'my-org/my-workspace' changes.
    bar = data.tfe_outputs.foo.values.bar
  }

  byte_length = 8
}

```

Reply:
fair point 👍